rules/pkg-normalize.js

```diff
@@ -22,6 +22,14 @@ const isGitRepository = repository => {
 		return false;
 	}
 
+	if (typeof repository !== 'string' && typeof repository !== 'object') {
```

Comment:
This check is independent of another task. This entire rule will break if `repository` is defined as an empty object because it will try to perform `.includes()` on an empty object `{}`. This means that if `repository` is invalid in these 2 cases

- It is not a `string` AND not an `object`
- It is an `object` but doesn't have a `url` property

The first check is covered, the second check isn't.

I also believe that the [pkg-schema](https://github.com/SamVerschueren/clinton/blob/master/rules/pkg-schema.js) rule checks for the validity of these properties. The extra check here would just make sure that the statement on line 33 works correctly and doesn't throw an exception.